sources.json

```diff
@@ -290,6 +290,34 @@
             }
         ]
     },
+    {
+        "shortOperatorName": "nrw-regional",
+        "contributors": [
+            {
+                "github": "metrophil"
+            }
+        ],
+        "sources": [
+            {
+                "name": "NRW-Regionalverkehrsplan 2023",
+                "source": "https://www.kcitf-nrw.de/fileadmin/03_KC_Seiten/KCITF/Regionalverkehrsplan/Regionalverkehrsplan_NRW_2023_Web.pdf"
+            }
+        ]
+    },
+    {
+        "shortOperatorName": "nrw-sbahn",
+        "contributors": [
+            {
+                "github": "metrophil"
+            }
+        ],
+        "sources": [
+            {
+                "name": "NRW-Fahrplanbuch 2023",
+                "source": "https://infoportal.mobil.nrw/information-service/fahrplanbuch-nrw.html"
+            }
+        ]
+    },
```

Comment:
I think that if you replace the RRX entries, the corresponding reference from the sources.json should also be removed.

Reply:
Done! :)